### PR TITLE
To fix error "unknown property"

### PR DESCRIPTION
**Pull request by githubjeka**
Getting unknown property: yii\grid\CheckboxColumn::attribute

### CODE REVIEW

**Comment by bupy7** [comment]
Could you give me example of this error?

**Comment by githubjeka** [comment]
```php
\bupy7\grid\GridView::widget([
        'layout' => "{summary}\n{pager}\n{items}\n{pager}",
        'id' => 'email-grid',
        'resizableColumns' => $resizableColumns,
        'resizableColumnsUrl' => $resizableColumnsUrl,
        'dataProvider' => $dataProvider,
        'filterModel' => $searchModel,
        'tableOptions' => ['class' => 'table table-striped'],
        'columns' => [
            [
                'class' => 'yii\grid\CheckboxColumn',
            ],
```

File: GridView.php [hunk 1]
```diff
@@ -434,6 +434,9 @@ public function initColumns()
         }
         if ($this->resizableColumns !== false) {
             foreach ($dataColumns as $column) {
+                if (!($column instanceof DataColumn)) {
```
Comment (githubjeka) [inline GridView.php:437]: May be better replace to
```php
!isset($column->attribute)
```

**Comment by bupy7** [comment]
What is in `$resizableColumns`?

**Comment by githubjeka** [comment]
```
array (size=4)
  'name' => string '72' (length=2)
  'language' => string '145' (length=3)
  'subject' => string '260' (length=3)
  'body' => string '356' (length=3)
```

**Comment by bupy7** [comment]
Hm... Very strange. It's column (`CheckboxColumn`) must be in `$serviceColumns` variable. I will check it later.

**Comment by githubjeka** [comment]
ERROR IS `Getting unknown property: yii\grid\CheckboxColumn::attribute`

See class `CheckboxColumn` and your code
```php
$column->headerOptions['data-resizable-column'] = $column->attribute;
```

**Comment by bupy7** [comment]
Yes, I understood you. But column `yii\grid\CheckboxColumn` must be in [`$serviceColumns`](https://github.com/githubjeka/yii2-grid/blob/3fc0d815a7e0b4e0699a9f9d1a9596ae3915f2c7/GridView.php#L428) and not in [`$dataColumns`](https://github.com/githubjeka/yii2-grid/blob/3fc0d815a7e0b4e0699a9f9d1a9596ae3915f2c7/GridView.php#L432).

**Comment by bupy7** [comment]
I understood! Don't you use visible columns functional, is it?

**Comment by githubjeka** [comment]
Yes

**Comment by bupy7** [comment]
That's right. You right.

File: GridView.php (hunk 1 above)
Comment (bupy7) [reply GridView.php:437]: http://php.net/manual/ru/language.operators.type.php#111527

Comment (githubjeka) [reply GridView.php:437]: I know, but with parentheses is more readable.

```php
if (!($column instanceof DataColumn)) {
```

```php
if (!$column instanceof DataColumn) {
```

Comment (bupy7) [reply GridView.php:437]: Ok. Don't worry. =)

**Pull request merged**